library/nist_kw.c

```diff
@@ -322,7 +315,7 @@ int mbedtls_nist_kw_wrap( mbedtls_nist_kw_context *ctx,
  * 1. Output buffer is at least of size( semiblocks - 1 ) * KW_SEMIBLOCK_LENGTH].
```

Comment:
There is an extra `]` before the `.`. Probably better to add a space before the `(` as well.

Reply:
Thanks it was an overmissed copy \ paste error